src/main/java/uk/gov/digital/ho/proving/income/IpsSearchPage.java

```diff
@@ -139,4 +146,10 @@ public WebElement getIncomeToDate() {
         return incomeToDate;
     }
 
+    public List<String> getEmploymentList() {
+        List<WebElement> employmentList = driver.findElements(By.cssSelector("li[ng-repeat='e in i.employers']"));
```

Comment:
This is the same as selector as the field called 'employmentList'. Isn't it possible to use that?

Reply:
I've removed the local employmentList and used the field one